pull-requests.md

```diff
@@ -0,0 +1,206 @@
+If you are not sure who to ask for help, get in touch with the `@core-yari-content` group.
+
+It is rare that you'll be required to review a large, complex content change with no warning, like a complete page rewrite, or the addition of several new reference pages or tutorials.
+Usually such changes are done as part of specific work streams where the content has been approved for addition and reviewer(s) have been assigned already.
+In such cases, the PR should be linked to an issue that explains all these details.
+If you are not sure, ask the PR author if they need a review of the content, and where the rationale behind the change is explained.
+Ping our team on MDN Web Docs chat room to ask for help if you are still not sure or if you think the content is suspicious.
+
+## Timelines
+
+This section provides details for expected turnaround times while reviewing PRs if you're a reviewer and while responding to review comments if you're a PR author.
+
+- PR review: The assigned reviewer should be able to review the PR in 2 weeks or less. In the 2 weeks after the PR is open, the assigned reviewer can:
+  - Leave a comment about when they can start reviewing the PR
+  - Ask for technical or resource help
+- Fixes for review comments: The PR author should be able to respond to or fix the comments in 4 weeks or less.
+  If the PR author is unable to respond or fix the review comments in that time, the reviewer can do one of the following:
+  - Commit the changes and merge the PR
+  - Close the PR
+
+## Topic owners
+
+Some of PRs submitted on the mdn/content repo relate to specific work streams being undertaken by browser vendors or other organizations that have a defined set of writers and reviewers. In these cases, the PR author of the PR will include the username of the reviewer in a line at the bottom of the pull request description, for example:
+
+reviewer: @jpmedley
+Upon submitting the PR, they will request a review from the reviewer specified in the PR description. Once that reviewer has approved the new content, they will then ask you for an approval as required by the CODEOWNERS system for the pull request to be mergeable.
+Therefore, if you receive a PR review request and then see that you have been overridden with another reviewer in the manner described above, then don't review the changes — just wait for an approval request.
+Guidelines for reviewing a pull request
+These guidelines apply to anyone who is tasked with reviewing MDN Web Docs content PRs.
+Prerequisite knowledge and Github tools
+
+The following specific topic areas are being reviewed by the kind souls listed underneath them. Be kind to them, and thank them for all the help they give to this project. If you would like to help with MDN content reviews, get in touch with us.
+Note that changes to any content areas not explicitly listed below will be handled by the @core-yari-content team, which currently consists of @Rumyra.
+
+- Web accessibility content:
```

Comment:
A table might be nice for this. It needs to be formatted differently, the list is too long / hard to read in this state

Reply:
Shall we remove it for now?